tools/firtool/firtool.cpp

```diff
@@ -538,15 +540,23 @@ processBuffer(MLIRContext &context, TimingScope &ts, llvm::SourceMgr &sourceMgr,
                                          nonConstAsyncResetValueIsError));
     pm.addPass(sv::createHWMemSimImplPass(replSeqMem, ignoreReadEnableMem));
 
-    if (extractTestCode)
-      pm.addPass(sv::createSVExtractTestCodePass());
-
-    // If enabled, run the optimizer.
-    if (!disableOptimization) {
-      auto &modulePM = pm.nest<hw::HWModuleOp>();
-      modulePM.addPass(createCSEPass());
-      modulePM.addPass(createSimpleCanonicalizerPass());
-      modulePM.addPass(sv::createHWCleanupPass());
+    if (outputFormat != OutputIRHW) {
```

Comment:
Minor nit: it might be more clear to flip the `if` and `else` here? Maybe this is just my reaction, but I tend to prefer positive conditions in `if`s.